packages/@expo/cli/e2e/__tests__/install-test.ts

```diff
@@ -100,3 +105,71 @@ it(
   // Could take 45s depending on how fast npm installs
   60 * 1000
 );
+
+it(
+  'runs `npx expo install --check` fails',
+  async () => {
+    const projectRoot = await setupTestProjectAsync('install-check-fail', 'with-blank');
+    await installAsync(projectRoot, ['add', 'expo-sms@1.0.0', 'expo-auth-session@1.0.0']);
+
+    let pkg = await JsonFile.readAsync(path.resolve(projectRoot, 'package.json'));
+    // Added expected package
+    expect(pkg.dependencies['expo-sms']).toBe('1.0.0');
+
+    try {
+      await execa('node', [bin, 'install', '--check'], { cwd: projectRoot });
+      throw new Error('SHOULD NOT HAPPEN');
+    } catch (error) {
+      expect(error.stderr).toMatch(/expo-auth-session@1\.0\.0 - expected version: ~3\.5\.0/);
+      expect(error.stderr).toMatch(/expo-sms@1\.0\.0 - expected version: ~10\.1\.0/);
+      expect(error.stderr).toMatch(
+        /npx expo install expo-auth-session@~3\.5\.0 expo-sms@~10\.1\.0/
+      );
```

Comment:
Which SDK version does the test project use? Would be great not to have to update this test every time we publish these packages 😛

Reply:
The fixtures are hardcoded against an SDK version, so it'll be part of updating the fixtures. May need to automate in the future, but for now the core functionality isn't expected to change across versions.